src/components/Router/routes.tsx

```diff
@@ -29,6 +30,26 @@ export const routesConfig: RouteObject[] = [
     path: RoutesPath.ABOUT,
     element: <About />,
   },
+  {
+    path: RoutesPath.DETAILS,
+    element: <LaunchDetailsPage />,
+  },
+  {
+    path: RoutesPath.PREVIOUS,
+    element: <Home />,
+  },
+  {
+    path: RoutesPath.CREWED,
+    element: <Home />,
+  },
+  {
+    path: RoutesPath.CALIFORNIA,
+    element: <Home />,
+  },
+  {
+    path: RoutesPath.FLORIDA,
+    element: <Home />,
+  },
```

Comment:
do you find a new approach?

Reply:
- added subroutes to `RoutesPath.HOME`
 - added LaunchList component, whice should be rendered  below th slider
 - added `<Outlet />` component from react-router-dom to markup in `Home.tsx`

Reply:
Good decision 👍